apps/ergw_core/src/gtp_context.erl

```diff
@@ -350,7 +351,8 @@ init([Socket, Info, Version, Interface,
       left_tunnel    => LeftTunnel,
       bearer         => Bearer},
 
-    Interface:init(Opts, Data).
+    {ok, State, LoopData} = Interface:init(Opts, Data),
```

Comment:
Adding some doc detailing that the current state is retrieved from somewhere would be really helpful.